app/views/docs/functions-execute.phtml

```diff
@@ -0,0 +1,513 @@
+<?php
+
+use Appwrite\Utopia\View;
+
+?>
+<p>
+    Appwrite Functions can be executed in several ways.
+    Executions can be invoked through the Appwrite SDK and visiting its REST endpoint. Functions can also be triggered by events and scheduled executions.
+    Here are all the different ways to consume your new Appwrite Functions.
+</p>
+
+<h2><a href="#domains" id="domains">Domains</a></h2>
+<p>
+    Each Appwrite function has its own domain. 
+    You can find this in the Appwrite Console, under the Function overview.
+</p>
+
+<p>    
+    The generated domains will look like this.
+</p>
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>https://64d4d22db370ae41a32e.appwrite.global</code></pre>
+</div>
+
+<p>
+    Alternatively you can <a href="/docs/functions-deploy#domain">add a custom domain</a> to your Appwrite project. 
+</p>
+
+<h2><a href="#rest" id="rest">REST API</a></h2>
+<p>
+    When requests are made to this domain, whether through a browser or through an HTTP requests, 
+    the request information like request headers and request body will be passed to the function.
+    This unlocks interesting ways to integrate other apps and backends to your Appwrite project.
+</p>
+
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre><code class="language-bash" data-prism>curl -X POST https://64d4d22db370ae41a32e.appwrite.global \
+    -H "X-Custom-Header: 123" \
+    -H "Content-Type: application/json" \
+    -d '{"data":"this is json data"}'</code></pre>
+</div>
+
+
+<h2><a href="#sdk" id="sdk">SDK</a></h2>
+<p>
+   You can invoke your Appwrite Functions directly from the Appwrite SDKs.
+</p>
+
+<p>
+<a href="/docs/sdk/functions"><i class="icon-angle-circled-right"></i>Learn more about using the Appwrite SDKs</a>
+</p>
+
+<h3>Client SDKs</h3>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div>
+        <div class="ide margin-top-small" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-js" data-prism>import { Client, Functions } from 'appwrite';  
+
+const client = new Client()
+
+client
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('[PROJECT_ID]')
+
+const functions = new Functions(client)
+
+try {
+    const data = await functions.createExecution('[FUNCTION_ID]', {
+        'foo': 'bar'
+    })
```

Comment:
```suggestion
    const data = await functions.createExecution('[FUNCTION_ID]', JSON.stringify({
        'foo': 'bar'
    }), '/', 'GET', {
      'X-Custom-Header': '123'
    })
```

Please make those changes to other languages as well. Body must be stringified JSON. Next params are `path`, `method` and `headers`. Everything is optional except function ID.

Reply:
Also do the same to server SDK. Same order of params, same rules.